sdk/spring/azure-spring-boot-test-aad/src/test/java/com/azure/test/aad/converter/RefreshTokenScopesIT.java

```diff
@@ -0,0 +1,88 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.test.aad.converter;
+
+import com.azure.test.oauth.SeleniumTestUtils;
+import com.azure.test.utils.AppRunner;
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity;
+import org.springframework.security.oauth2.client.OAuth2AuthorizedClient;
+import org.springframework.security.oauth2.client.annotation.RegisteredOAuth2AuthorizedClient;
+import org.springframework.security.oauth2.core.OAuth2AccessToken;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.*;
+
+public class RefreshTokenScopesIT {
+
+    @Test
+    public void testRefreshTokenConverter() {
+        try (AppRunner app = new AppRunner(DumbApp.class)) {
+            SeleniumTestUtils.addProperty(app);
+            app.property("azure.activedirectory.authorization.office.scopes", "https://manage.office.com/ActivityFeed.Read");
+            app.property("azure.activedirectory.authorization.graph.scopes", "https://graph.microsoft.com/User.Read");
+            List<String> endPoints = new ArrayList<>();
+            endPoints.add("api/office");
+            endPoints.add("api/azure");
+            endPoints.add("api/graph");
+            endPoints.add("api/arm");
+            Map<String, String> result = SeleniumTestUtils.get(app, endPoints);
+
+            Assert.assertFalse(result.get("api/office").contains("profile"));
```

Comment:
Why `office` and `graph` should contain `profile` scope? I remember `openid` and `profile` only needed for on-demand client.